docs/configuration.md

```diff
@@ -376,6 +402,30 @@ parameter1: Type1 // indented by 4
 ): ReturnType
 ```
 
+### `indent.binPackDefnSite`
+
+This parameter controls indentation of method parameters when they are
+binpacked (that is, spanning multiple lines but not formatted using config
+style). Defaults to `indent.defnSite`.
+
+> Since v3.8.3.
+
+Example:
+
+```scala mdoc:scalafmt
+newlines.configStyleDefnSite.prefer = false
+danglingParentheses.defnSite = false
+binPack.defnSite = always
+indent.defnSite = 2
+indent.binPackDefnSite = 4
```

Comment:
Why does it need to be separate to `indent.defnSite`, that seems to over complicate things

Reply:
how would you propose to do it? the have a requirement to use one indent when config style formatting is used, and a different one otherwise. which seems to mean one parameter is not enough...

Reply:
It's probably not possible any other way, but I don't see how this kind of option is useful for anyone to have some many different indents

Reply:
Could you point me to the requirement, where it's defined?

Reply:
Nvm. I found it, but I am still not convinced it's more than just an OCD thing and doesn't really help in readability that much.

Reply:
i think the reason it's useful to scala.js developers is that it visually distinguishes between clean config style formatting, and more readability-challenged binpack formatting. one of the reasons scala.js has not adopted scalafmt yet.